modules/claims_api/spec/sidekiq/poa_vbms_updater_spec.rb

```diff
@@ -54,18 +51,29 @@
       let(:consent_address_change) { nil }
 
       it 'updates a the BIRLS record for a qualifying POA submittal' do
-        poa = create_poa
+        poa = create_poa(allow_poa_access: true)
         create_mock_lighthouse_service
         subject.new.perform(poa.id)
       end
     end
 
+    context "when allow_poa_access is set to 'n'" do
+      let(:allow_poa_c_add) { 'N' }
+      let(:consent_address_change) { nil }
+
+      it 'does not allow folder access' do
+        poa = create_poa(allow_poa_access: false)
+        create_mock_lighthouse_service_no_access
+        subject.new.perform(poa.id)
+      end
+    end
```

Comment:
I think I may have misunderstood how they were running.  I thought the assertions where down in the methods.

Reply:
oh, i see now. yes, the assertions are in the below methods 😅. i think the other methods were meant to test that the services are working. do the assertions in `create_mock_lighthouse_service_no_access` reflect the test description `does not allow folder access`? from what i can tell, it seems we are testing that the call works.

Reply:
Well the call should still work, just send a 'N' so eFolder access is turned off.  To be honest I'm not sure I feel this test is really needed.  I added tests for the method in the service_base to handle making sure it returns the values as expected, and those I think are good for this, I added this to avoid having it appear like something was getting missed but I'm not a huge fan of this test file.  That said, the call should still succeed, just _not_ enable eFolder access

Reply:
i agree the test is not needed. if we want to keep it, should we change the description to something more fitting, like “the call succeeds”? as written, it seems like we are testing whether the folder can be accessed after setting the value to N.